### PR TITLE
Revert "To fix ios_bgp_address_family neighbor next_hop_self param"

### PR DESCRIPTION
**Pull request by KB-perByte**
Reverts ansible-collections/cisco.ios#323

### CODE REVIEW

File: plugins/module_utils/network/ios/facts/bgp_address_family/bgp_address_family.py [hunk 1]
```diff
@@ -33,6 +35,16 @@ class Bgp_AddressFamilyFacts(object):
     def __init__(self, module, subspec="config", options="options"):
         self._module = module
         self.argument_spec = Bgp_AddressFamilyArgs.argument_spec
+        spec = deepcopy(self.argument_spec)
```
Comment (justjais) [inline plugins/module_utils/network/ios/facts/bgp_address_family/bgp_address_family.py:38]: these lines are not related to next_hop_self revert, had removed these lines to increase TC coverage

**Review by justjais — CHANGES_REQUESTED** [review]

**Comment by KB-perByte** [comment]
Closing as PR has parts for generic operation, 
#361 added
